src/orchard/misc.clj

```diff
@@ -41,9 +41,11 @@
                     (when  (= (.getProtocol ^java.net.URL f) "file")
                       (io/as-file f))
                     (io/as-file f))]
-    (some (fn [ext]
-            (.endsWith (.. file getName toLowerCase) ext))
-          exts)))
+    (and (some (fn [ext]
```

Comment:
Doesn't it make more sense to swap the order of changes - check first that something is not a folder? That reads better to me, and probably won't have much impact on performance.